scipy/integrate/quadrature.py

```diff
@@ -375,6 +382,23 @@ def simps(y, x=None, dx=1, axis=-1, even='avg'):
     the samples are not equally spaced, then the result is exact only
     if the function is a polynomial of order 2 or less.
 
+    If using method 'extended' and number of samples is less than 8
+    they must be even. 
+    If using method 'second' number of samples must be of the form 3*k+1 
+    
+    N = Number of samples.
+    Formula used in 'composite' method when number of samples is odd:-
+    Integral = dx/3*[y[0] + 4*y[1] + 2*y[2] + 4*y[3]..... +4*y[N-2] + y[N-1]]
+
+    Formula used in 'second' method:-
+    Integral = 3*dx/8*[y[0] + 3*y[1] + 3*y[2] + 2*y[3] + 3*y[4] + 3*y[5] + 2*y[6]...+ y[N-1]]
+
+    Formula used in 'extended' method:-
+    Integral = dx/48*[17*y[0] + 59*y[1] + 43*y[2] + 49*y[3] + 48*X + 49*y[N-4] + 43*y[N-3]
+                     + 59*y[N-2] + 17*y[N-1]]
+    where X = (from i = 4 to i = N-5 ) Σ y[i]
```

Comment:
IMO formulas of this complexity are best written in TeX notation.

```
.. math::

  \frac{dx}{48} \left(  17 y_0 + 59 y_1 + 43 y_2 + 49 y_3 ... \right)
```

(Depending on how you write it, TeX can be anywhere between crystal clear and utterly unreadable).

In any case, unicode symbols are best avoided. Either use pseudo-python pseudocode, `sum(y[i] for i in range(4, N-4))` or TeX: `\sum_{i=4}{N-5} y_i`